frontend/src/components/About.js

```diff
@@ -5,16 +5,41 @@ import { BACKEND_URL } from '../utils/api'
 import SectionHeader from './utils/SectionHeader'
 import './About.scss'
 
+function countString(str, ch) {
+  let count = 0
+    for (let i = 0; i < str.length; i++) {
+      if (str.charAt(i) === ch) {
+        count += 1
+      }
+    }
+  return count
+}
+
 const Bio = ({profile, avatar}) => {
+  while (countString(profile.name, ' ') !== 1) {
+    profile.name = profile.name.slice(profile.name.indexOf(' ') + 1)
+  }
```

Comment:
Is this logic required by Figma em oi? Looks overkill.

Reply:
This is the design on Figma íi anh
<img width="781" alt="Screen Shot 2022-06-16 at 15 35 52" src="https://user-images.githubusercontent.com/87293327/174029168-2ca13b50-20ec-4e46-8112-e6aac51d55bd.png">